src/main.rs

```diff
@@ -64,6 +65,15 @@ fn main() {
                     matches.value_of("version")
                 ),
             ("show", Some(_)) => show_cmd::run(),
+            ("doctor", Some(matches)) => {
+                let fix = match matches.value_of("fix_mode") {
+                    Some("auto") => doctor_cmd::FixMode::Auto,
+                    Some("never") => doctor_cmd::FixMode::Never,
+                    Some("prompt") => doctor_cmd::FixMode::Prompt,
+                    _ => unreachable!()
+                };
+                doctor_cmd::run(fix)
```

Comment:
This should probably be named `fix_mode`. The `fix` name comes from how it was implemented in an early iteration